circle.yml

```diff
@@ -18,38 +18,28 @@ dependencies:
     - cp version.json $CIRCLE_ARTIFACTS
     - docker info
     # use circleci's docker cache workaround
+    - mkdir -p ~/cache/
     - if [ -e ~/cache/docker/image.tar ]; then echo "Loading image.tar"; docker load -i ~/cache/docker/image.tar || rm ~/cache/docker/image.tar; fi
     # build image
     - docker build -t morgoth:build .
-    # Clean up old image and save the new one
-    - mkdir -p ~/cache/docker; test '!' -e ~/cache/docker/image.tar || rm ~/cache/docker/image.tar; docker save morgoth:build > ~/cache/docker/image.tar; ls -l ~/cache/docker
+    # write the sha256 sum to an artifact to make image verification easier
+    - docker inspect -f '{{.Config.Image}}' morgoth:build | tee $CIRCLE_ARTIFACTS/docker-image-shasum256.txt
```

Comment:
Turns out this isn't actually useful to our ppsfolk, so I wouldn't bother adding this (and it should be removed from Normandy).